components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -1219,11 +1230,11 @@ subroutine radiation_tend(state,   ptend,    pbuf,          cam_out, cam_in,  &
          call handle_error(clip_values( &
             tmid(1:ncol,1:nlev_rad), k_dist_lw%get_temp_min(), k_dist_lw%get_temp_max(), &
             trim(subname) // ' tmid' &
-         ), fatal=.false.)
+         ), fatal=.false., warn=rrtmgp_enable_limiter_warnings)
```

Comment:
It doesn't look like there's a corresponding change for this in the MMF version of this file, so these changes have only been applied to the non-MMF radiation interface. Since you've added the namelist machinery there, please make the corresponding change to the call to `clip_values` in components/cam/src/physics/crm/rrtmgp/radiation.F90 as well.

Reply:
oh, I thought I already did this. Maybe I just forgot to commit it. I'll add it.